app/services/DateService.scala

```diff
@@ -46,10 +46,19 @@ class DateService @Inject()(implicit val frontendAppConfig: FrontendAppConfig) e
       case _ => currentDate.getYear + 1
     }
   }
+
+  def getCurrentTaxEndYear: Int = {
```

Comment:
this need to be reduced to =>
getCurrentTaxYearEnd(getCurrentDate)

Reply:
Removed getCurrentTaxEndYear as we already had a method returning the same thing